src/tests/unit/handler.test.js

```diff
@@ -3,18 +3,50 @@
 const { handler } = require('../../handler')
 const chai = require('chai')
 const expect = chai.expect
+const httpStatus = require('http-status-codes')
 
-describe('Handler throws error when called', () => {
-  it('verifies response is error 500 and  has Internal Server Error message', async function () {
+describe('Handler throws error when called with illegal path', () => {
+  it('verifies response is error 404-NOT FOUND, and has Not Found error message', async function () {
     const calledPath = '/some.path'
     const event = { path: calledPath }
     const response = await handler(event)
-    expect(response.statusCode).to.equal(500)
+    expect(response.statusCode).to.equal(httpStatus.NOT_FOUND)
     const responseBody = JSON.parse(response.body)
     expect(responseBody.instance).to.equal(calledPath)
-    expect(responseBody.status).to.equal(500)
-    expect(responseBody.detail).to.equal("Your request cannot be processed at this time due of 'Internal Server Error'")
-    expect(responseBody.title).to.equal('Internal Server Error')
+    expect(responseBody.status).to.equal(httpStatus.NOT_FOUND)
+    expect(responseBody.detail).to.equal("Your request cannot be processed at this time due of 'Not Found'")
+    expect(responseBody.title).to.equal(httpStatus.getReasonPhrase(httpStatus.NOT_FOUND))
     expect(responseBody.type).to.equal('about:blank')
   })
 })
+
+describe('Handler throws error when called with illegal method and correct path', () => {
```

Comment:
Handler throws error when input is an http-request with illegal method. ( and correct path) is not necessary, because we assume that we always test one dimension.